ios/packages/reference-assets/Sources/SwiftUI/CollectionAsset.swift

```diff
@@ -31,6 +31,7 @@ struct CollectionAssetView: View {
     @ViewBuilder
     var body: some View {
         VStack {
+            model.data.label?.asset?.view.font(.label)
```

Comment:
This has a [compilation error](https://app.circleci.com/pipelines/github/player-ui/player/1234/workflows/8e44c482-1e9c-4333-a6a8-955ce643e1f6/jobs/6084) as `CollectionData` has no `label` member. You will need to update the data struct to reflect what needs to be decoded